### DISCUSSION TITLE
Some one hacked my GitHub Profile

### DISCUSSION THREAD

**Discussion by Musawar71** [community/community#120598] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Some one has hacked my github Profile engrmusawarali they added their email. Previously "engrmusawar@gmail.com" was associated with this account

**Comment by ettaboyle** [comment, marked as answer]
Hi @Musawar71, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by Guhvyrf** [comment]
Hack we id
